### ISSUE TITLE
editorconfig-emacs failed to build on Ubuntu 14.04

### ISSUE THREAD

**Issue by rwhogg** [Homebrew/homebrew-emacs#129]
editorconfig-emacs no longer builds after [this commit](https://github.com/Homebrew/homebrew-emacs/commit/476b71531199c7660696c21f7eec4bd331754e20) bumped it to version 0.6.1.

The gist is [here](https://gist.github.com/cadf3414f6f69a533523)

**Comment by dunn** [comment]
Interesting, thanks.  Have the commits you cherry-picked in #54 been pulled into Linuxbrew, or is that still pending?

**Comment by rwhogg** [comment]
Yeah, they've both been merged.

BTW, if you'd like some help testing commits to this tap on Linux going forward, I'd be happy to lend a hand.

**Comment by dunn** [comment]
That'd be great!  I should also set up the Travis config to run tests on Linux; if you want to take a stab at that I'd be :heart_eyes_cat:

**Comment by dunn** [comment]
I remembered I have a Debian VM on Digital Ocean, and I'm seeing the same failure there.

**Comment by rwhogg** [comment]
I'll get to work on setting up Travis.

**Comment by rwhogg** [comment]
I made PR #131 to set up tests on Linux.

One thing: I noticed that in both my Linux build and some of the existing OS X builds, we see that bundler is not installed, but it's getting called by the Travis script. Does that matter?

**Comment by rwhogg** [comment]
I stand corrected about the two Emacs fixes getting into Linuxbrew. They were there, but I forgot about https://github.com/Homebrew/linuxbrew/commit/787d7f9be79ec46cf4614573993fc952a73c41c1#diff-78ce57246c1574f0ecfe4e82586ff452, which removed it again.

**Comment by dunn** [comment]
Beacon and GH.el are also failing on Linux, and looks like the common issue is seq.el:
- beacon logs https://gist.github.com/dunn/0759928e64a7692ee782
- gh logs https://gist.github.com/anonymous/d3b63591db60e3738ce2

**Comment by dunn** [comment]
Not the real issue, though; reinstalling seq fixed those build errors, but `make test` is still failing here.

**Comment by rwhogg** [comment]
I'm going to try modifying the formula to see if 0.6 (as opposed to 0.6.1) works. Will post results.

**Comment by dunn** [comment]
Think I found the issue. See if #132 works for you.

**Comment by rwhogg** [comment]
Yeah, that commit fixes it. Thanks a bunch!

**Comment by dunn** [comment]
Any idea why `$(PWD)` is evaluated as the directory from which `brew install` is run on Linux, instead of the directory from which `make` is run?

**Comment by rwhogg** [comment]
Not off the top of my head, unfortunately. My apologies.

**Comment by rwhogg** [comment]
```sh
Script started on Sat 12 Dec 2015 12:31:21 PM PST
☁  ~  irb
irb(main):001:0> Dir.pwd
=> "/home/bob"
irb(main):002:0> Dir.chdir "/home/bob/src"
=> 0
irb(main):003:0> Dir.pwd
=> "/home/bob/src"
irb(main):004:0> ENV["PWD"]
=> "/home/bob"
irb(main):005:0> 
☁  ~  
Script done on Sat 12 Dec 2015 12:31:49 PM PST
```

Based on a quick irb session, I'm going to guess that it's because Ruby has its own notion of what the current working directory is and it doesn't update the environment variables when people call `Dir.chdir` or the like. So ENV["PWD"] is going to resolve to whatever $(PWD) was at the time we spawned the ruby executable, i.e. where we ran `brew install` from. If we were to spawn a shell from within Ruby, I assume it would also inherit the environment, again making $(PWD) where we originally launched from.

Not 100% sure; my knowledge of Ruby is next-to-nil.
